frontend/lib/verification/scanner/verification_encoder.dart

```diff
@@ -11,7 +11,7 @@ String encodeVerificationCardDetails(VerificationCardDetails verificationCardDet
   final verifyCode = CardVerifyCode(
     info: CardInfo(
       fullName: cardDetails.fullName,
-      expiration: Expiration(day: cardDetails.expirationDay ?? 0),
+      expirationDay: cardDetails.expirationDay ?? 0,
```

Comment:
This looks wrong to me. I think we should set it to null here (or omit it or so). Otherwise we explicitly set the value of `expirationDay` to `0` (and it is no longer marked as "absent")